src/components/screens/votingQueue/editor/editor.js

```diff
@@ -158,60 +138,75 @@ const Editor = ({
     });
   };
 
+  const showEmptyState = !changedVotes.length;
+
   return (
     <section className={styles.wrapper}>
       <Box>
         <ToggleIcon isNotHeader />
-        <VoteStats
-          t={t}
-          heading={t('Voting queue')}
-          added={Object.keys(added).length}
-          edited={Object.keys(edited).length}
-          removed={Object.keys(removed).length}
-        />
-        <BoxContent className={styles.contentContainer}>
-          <div className={styles.contentScrollable}>
-            <Table
-              data={changedVotes}
-              header={header(t)}
-              row={VoteRow}
-              iterationKey="address"
-              canLoadMore={false}
-              emptyState={{ message: t('No votes in queue.') }}
+        <div className={styles.headerContainer}>
+          <header>
+            {t('Voting Queue')}
+          </header>
+          {!showEmptyState ? (
```

Comment:
I prefer ```!showEmptyState &&``` . Then it won't need ``` : null}``` which will make the code cleaner

Reply:
https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Reply:
I knew about this problem but showEmptyState is a boolean and should be fine. I applied the requested change, but if @UsamaHameed disagrees I can revert it